### ISSUE TITLE
Finding the optimal hyperparameters

### ISSUE THREAD

**Issue by saipriya0209** [megankaym/1015finalproject#4]
If the accuracy is not the most optimal/best for this model, maybe a hyperparameter fine-tuning step can be added to get the best possible accuracy.
